modular_citadel/code/modules/reagents/reagents/cit_reagents.dm

```diff
@@ -37,10 +37,11 @@
 	..()
 	dir = pick(1,2,4,8)
 
-/datum/reagent/consumable/semen/reaction_turf(turf/T, reac_volume)
+/* /datum/reagent/consumable/semen/reaction_turf(turf/T, reac_volume)
```

Comment:
Just delete this commented code, it serves no purpose anymore.

Reply:
Aite.

Reply:
I found out that was originally from a merge conflict that was resolved incorrectly. Just pennies for thought.